### ISSUE TITLE
Scrolling under the General Settings tab stops at 'Autostart'

### ISSUE THREAD

**Issue by anoadragon453** [qTox/qTox#2590]
Hi there, using the latest qTox build on Mac OSX 10.11.1, based off 4d54f5c.

When two-finger scrolling through the general tab, I am unable to scroll once the mouse hovers over the disabled (greyed-out) Autostart feature.

<img width="509" alt="screen shot 2015-11-25 at 2 02 13 am" src="https://cloud.githubusercontent.com/assets/1342360/11394228/9b26a984-9318-11e5-817d-2abc39fa6c36.png">

Unsure if this affects other platforms.

Thanks.

**Comment by ghost** [comment]
This was apparently fixed for other OSs in #2538
Related: #2302 #2536

**Comment by anoadragon453** [comment]
Seems like it's still present in @RowenStipe's build. I don't have the resources to build the latest code with me at the moment and check if it's been fixed, but his was built 5 days after that PR was merged.

**Comment by TheNain38** [comment]
I don't know how Qt handles EventFilters on MAC OSX.
@anoadragon453 Could you tell me in the audio/video tab if you can change sliders position by scrolling on them. If you can it may means that Qt can't use EventFilters on MAC OSX

**Comment by anoadragon453** [comment]
I can change them but only by scrolling horizontally. When I do so the vertical scrolling does not lock, and thus it slightly scrolls vertically as well.

**Comment by TheNain38** [comment]
Are you able to change the value of combo boxes by scrolling on them?

**Comment by anoadragon453** [comment]
Scrolling on the Language box on the General tab for example scrolls the entire screen. AKA it does nothing.

**Comment by RowenStipe** [comment]
Is this still a bug? Attempted to reproduce using a more recent commit: 0ef39aa06af177061ab53c259ee9c71de2876264 and it's not happening anymore with any of the `Style` options in the `General` tab or with a detached window using a mouse with a scroll-wheel or trackpad.

**Comment by anoadragon453** [comment]
I'm using your latest "PATCH-1.0.1.2" build on OSX 10.11.1 and is still an issue. I have only tested with the trackpad but I don't imagine a mouse will cause much difference.

**Comment by RowenStipe** [comment]
After trying to reproduce https://github.com/tux3/qTox/issues/2591 with no sucess using the same build you are using. I'm wondering if both issues might be an issue with 10.11.1 OSX since I've been using 10.10.5 to build.

I've been hesitant on upgrading to El Capitan since I'm using a MacBook Pro with slightly better specs than my smartphone. So this is just a theory right now.

**Comment by anoadragon453** [comment]
It's definitely possible. I could see if I can find an OSX 10.10 VM to throw up and see if it's still an issue.

**Comment by RowenStipe** [comment]
If you can't get ahold of a 10.10 image let me know, I'm currently at 3 GB downloaded for the Yosemite Installer (which has the image).

**Comment by anoadragon453** [comment]
I have a 10.11 image set up already - I'll test on there for the issues and see if they occur once I get home.

**Comment by RowenStipe** [comment]
Alright!

**Comment by anoadragon453** [comment]
Alright, so, I wasn't able to reproduce the issues on the VM. Interestingly enough, I did notice the qTox interface seemed a bit more updated than the one I had on my computer. 

Low and behold, I was testing an old version.

It seems your .DMG links Applications to ~/Applications, while most other .DMGs link it to <HDD Name>/Applications (for whatever reason). Thus opening qTox from the latter folder was still the old release.

That being said, I have found the 'Autostart' scrolling issue is no longer reproducible! However, the scrolling one still is. I've included a gfy demo over on that issue.
